src/synth/fluid_synth.c

```diff
@@ -555,6 +549,8 @@ new_fluid_synth(fluid_settings_t *settings)
   double gain;
   int i, nbuf;
   int with_ladspa = 0;
+  int with_reverb;
+  int with_chorus;
```

Comment:
I would initialize with_reverb and with_chorus explicity. Otherwise compilers might complain about a possible unitialized use of those variables...